### PR TITLE
OSSM-5302 Use profiles 'default' and 'openshift' as the default on OpenShift

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: bundle/manifests/sailoperator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -363,6 +362,7 @@ spec:
               - args:
                 - --health-probe-bind-address=:8081
                 - --metrics-bind-address=127.0.0.1:8080
+                - --default-profiles=default,openshift
```
Comment (jwendell) [inline bundle/manifests/sailoperator.clusterserviceversion.yaml:365]: I'm confused. This deployment is also used in kubernetes if OLM is used, no?

Comment (luksa) [reply bundle/manifests/sailoperator.clusterserviceversion.yaml:365]: Does anyone really run OLM in k8s? Does it even work?

Comment (luksa) [reply bundle/manifests/sailoperator.clusterserviceversion.yaml:365]: I mean.. does anyone run the sail operator via OLM in Kubernetes.

Comment (jwendell) [reply bundle/manifests/sailoperator.clusterserviceversion.yaml:365]: Not yet. We have an issue created for that. To publish our operator in general operatorhub (not the current, ocp one).

Comment (luksa) [reply bundle/manifests/sailoperator.clusterserviceversion.yaml:365]: So, that sounds the correct time to figure out how to properly configure the operator according to the underlying platform.

**Review by jwendell — APPROVED** [review]

**Pull request merged**